src/app/public/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -116,11 +116,6 @@ describe('File attachment', () => {
 
     const imageEl = getImage();
     expect(imageEl.nativeElement.getAttribute('src')).toBe('myFile.' + extension);
-
-    // Test Accessibility
-    fixture.whenStable().then(() => {
```

Comment:
These async tests are having trouble completing in a loop, with the latest version of Jasmine. I'm not even sure why we're running them here, since we have a dedicated a11y check later on.